src/main/java/com/github/tomakehurst/wiremock/matching/ContentPatternDeserializer.java

```diff
@@ -16,20 +16,19 @@
 package com.github.tomakehurst.wiremock.matching;
 
 import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
-import com.fasterxml.jackson.databind.JsonMappingException;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.ImmutableList;
 import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
 import java.util.Map;
 
-public class ContentPatternDeserialiser extends JsonDeserializer<ContentPattern<?>> {
+public class ContentPatternDeserializer extends JsonDeserializer<ContentPattern<?>> {
```

Comment:
Not sure @tomakehurst will like that :) Let's keep the British English for now

Reply:
Done